### PR TITLE
fix: blobscan redis dependency

### PR DESCRIPTION
**Pull request by barnabasbusa**

### CODE REVIEW

File: src/blobscan/blobscan_launcher.star [hunk 1]
```diff
@@ -184,11 +211,12 @@ def get_web_config(
...
-            "SECRET_KEY": "supersecret",
-            "NEXT_PUBLIC_NETWORK_NAME": "devnet",
-            "BEACON_NODE_ENDPOINT": beacon_node_rpc,
-            "CHAIN_ID": chain_id,
+            "DATABASE_URL": postgres_url,
+            "NEXT_PUBLIC_NETWORK_NAME": network_name
+            if network_name in constants.PUBLIC_NETWORKS
+            else "devnet",
+            "NEXT_BEACON_NODE_ENDPOINT": beacon_node_rpc,
+            "EXECUTION_NODE_ENDPOINT": execution_node_rpc,
```
Comment (PJColombo) [inline src/blobscan/blobscan_launcher.star:219]: This env vars are only required by the indexer.

The web app doesn't use them.

**Review by PJColombo — COMMENTED** [review]
Left one comment regarding some unused env vars.

Other than that, it LGTM!

File: src/blobscan/blobscan_launcher.star (hunk 1 above)
Comment (barnabasbusa) [reply src/blobscan/blobscan_launcher.star]: Removed.

**Comment by barnabasbusa** [comment]
We are back in the game. 

Screenshot 2024-07-29 at 13 44 58

**Review by parithosh — APPROVED** [review]

**Pull request merged**